src/views/Votes.vue

```diff
@@ -125,128 +125,155 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
       page: 1
-    },
-    expanded: [],
-    waitingForConfirmation: false,
-    dialog: false
-  }),
-  computed: {
-    className: () => 'delegates-view',
-    delegates () {
-      const delegates = this.$store.state.delegates.delegates || {}
+    })
+    const expanded = ref([])
+    let waitingForConfirmation = ref(false)
+    let dialog = ref(false)
+    const className = 'delegates-view'
+
+    const delegates = computed(() => {
+      const delegates = store.state.delegates.delegates || {}
 
       return Object.values(delegates)
-    },
-    numOfUpvotes () {
-      return this.delegates
-        .filter(delegate => delegate.upvoted && !delegate.voted)
-        .length
-    },
-    numOfDownvotes () {
-      return this.delegates
-        .filter(delegate => delegate.downvoted && delegate.voted)
-        .length
-    },
-    totalVotes () {
-      return this.delegates.filter(delegate => delegate._voted).length
-    },
-    pages () {
-      if (this.delegates.length <= 0) {
+    })
+    const numOfUpvotes = computed(() => {
+      return delegates.value
+          .filter(delegate => delegate.upvoted && !delegate.voted)
+          .length
+    })
+    const numOfDownvotes = computed(() => {
+      return delegates.value
+          .filter(delegate => delegate.downvoted && delegate.voted)
+          .length
+    })
+    const totalVotes = computed (() => {
+      return delegates.value.filter(delegate => delegate._voted).length
+    })
+    const pages = computed(() => {
+      if (delegates.value.length <= 0) {
         return 0
       }
 
-      return Math.ceil(this.delegates.length / this.pagination.rowsPerPage)
-    },
-    showPagination () {
-      return this.search.length === 0
-    },
-    reviewButtonDisabled () {
-      return (this.numOfUpvotes + this.numOfDownvotes) === 0
-    }
-  },
-  mounted () {
-    this.$store.dispatch('delegates/getDelegates', {
-      address: this.$store.state.address
+      return Math.ceil(delegates.value.length / pagination.rowsPerPage)
     })
-  },
-  methods: {
-    getExplorerUrl () {
+    const showPagination = computed(() => {
+      return search.value.length === 0
+    })
+    const reviewButtonDisabled = computed(() => {
+      return (numOfUpvotes.value + numOfDownvotes.value) === 0
+    })
+
+    onMounted(() => {
+      store.dispatch('delegates/getDelegates', {
+        address: store.state.address
+      })
+    })
+
+    const getExplorerUrl = () => {
       return explorerUriToOnion('https://explorer.adamant.im/delegate/')
-    },
-    upVote (id) {
-      this.$store.commit('delegates/upVote', id)
-    },
-    downVote (id) {
-      this.$store.commit('delegates/downVote', id)
-    },
-    sendVotes () {
-      if (!this.validateVotes()) {
+    }
+    const upVote = (id) => {
+      store.commit('delegates/upVote', id)
+    }
+    const downVote = (id) => {
+      store.commit('delegates/downVote', id)
+    }
+    const sendVotes = () => {
+      if (!validateVotes()) {
```

Comment:
The `validateVotes()` is called before declaration. Swap `sendVotes` and `validateVotes` function declarations.